### ISSUE TITLE
pants version not detected corrected in non-conventional pants.toml location 

### ISSUE THREAD

**Issue by clsx524** [pantsbuild/scie-pants#153]
Hi, I am trying out the SCIE pants and specify pants 2.16.0rc0 in pants.toml. This file is put under a subdirectory of my repo so before I have an env var PANTS_TOML to indicate that. however I found this is not going to work. If I move pants.toml to the top level it would work.

The problem I am seeing is
```
The `pants.toml` at /Users/xxxx/xxxx/xxxxx/pants.toml has no `pants_version` configured in the `GLOBAL` section.
Would you like set `pants_version` to the latest stable release? (Y/n): 
```

and if I select no, it would crash with
`ERROR: Failed to establish atomic directory /Users/xxxx/Library/Caches/nce/142382d6e3ac0960e7d4f81b9df9de9559b7ac698db13d20cbd0e4aa57f68c1d/locks/configure-a04b16475909adc53343e9b2a72885b57d783b5743b4fbcdad57d8d31e81ace7. Population of work directory failed: Boot binding command failed: exit status: 1`

if I select yes, it would update my pants.toml to 2.15.0

**Comment by jsirois** [comment]
Thanks for pointing this out @clsx524. I now see I missed this feature in the `./pants` script when porting to scie-pants:
https://github.com/pantsbuild/setup/blob/15988fe15cc87031235ecd0e94ebc38df692a910/pants#L36-L39

**Comment by jsirois** [comment]
I'll not @clsx524 you will need to mark your build root still. Now that the `./pants` script is gone, that cannot be the marker any longer and so you'll need to create an empty `BUILD_ROOT` file at the root of your repo and check that in.

**Comment by jsirois** [comment]
@clsx524 the fix is released in [scie-pants 0.6.0](https://github.com/pantsbuild/scie-pants/releases/tag/v0.6.0). You can run `SCIE_BOOT=update pants` to grab the update.
